DNS/Protocol/ResourceRecords/SrvResourceRecord.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Net;
+
+namespace DNS.Protocol.ResourceRecords
+{
+    public class SrvResourceRecord : BaseResourceRecord
+    {
+        private static IResourceRecord Create(
+            Domain service,
+            TimeSpan ttl,
+            int priority,
+            int weight,
+            int port,
+            Domain target
+        )
+        {
+            var data = new List<byte>();
+
+            data.AddRange(BitConverter.GetBytes((ushort) priority).Reverse());
+            data.AddRange(BitConverter.GetBytes((ushort) weight).Reverse());
+            data.AddRange(BitConverter.GetBytes((ushort) port).Reverse());
+            data.AddRange(target.ToArray());
+
+            var type = RecordType.SRV;
+
+
+            return new ResourceRecord(service, data.ToArray(), type, RecordClass.IN, ttl);
+        }
+
+        public SrvResourceRecord(IResourceRecord record) : base(record)
+        {
+            //parse record
+            Service = Name;
```

Comment:
Don't know if I'm missing something, but where is `Name` coming from. And what about parsing the rest of the parameters from the binary data?

Reply:
is been some time since i wrote this but that is all that is need from what i remember ?

Reply:
If you parse it right now, all the fields won't be populated. You check the other resource record implementations on how they parse the binary data.

Reply:
no from what i remember(correct me if im wrong) that is all the fields the srv record is very very simple

Reply:
im using this code for service discover and it totaly works